src/utils/getPersonWithParents.tsx

```diff
@@ -0,0 +1,19 @@
+import { Person } from '../types';
+
+export const getPersonWithParents = (peopleFromServer: Person[]) => {
```

Comment:
You have the same function twice
[getPersonWithParents.tsx](https://github.com/mate-academy/react_people-table-advanced/pull/698/files#diff-35269717811afe57a5034e52a99ccfd5b8ef5fba35be9484bc1e9f90ced5718c)